frontend-cert/css-projects/blog-post-card/user-stories.md

```diff
@@ -0,0 +1,20 @@
+1. You should have a `div` with a class of `blog-post-card` to hold all your card elements. Step 2 and 3 should be created within the `blog-post-card` `div`.
+
+2. You should have an image with a class of `post-img`.
+
+3. You should have a `div` with a class of `post-content`. Within the `post-content` `div`:
+    - You should have a `h2` with a class of `post-title`. Assign some text to act as the title of your blog post.
+    - You should have a `p` with a class of `post-excerpt`. Assign some text to summarize the blog post (maximum of 20 words).
+    - You should an `a` with a class of `read-more`. Assign the text `Read More` to it.
+
+4. The `blog-post-card` class should add a white background, rounded corners, shadow for depth, overflow hidden to round the image borders, and control the card width and text alignment.
+
+5. The `post-img` class should ensure that the image fills the card's width and adds a border at the bottom.
+
+6. The `post-content` class should add padding inside the card for spacing.
+
+7. The `post-title` and `post-excerpt` elements within `post-content` should be styled for text color and margin.
+
+The `read-more` element should be styled for text color, background color, margin, `inline-block` display, `border-radius`, and `padding`. It should be styled like a button.
+
+9. The `read-more` element should have a lighter color when hovered on.
```

Comment:
Just wondering how this can be tested :thinking: 
Maybe it's enough to say a different color?

Reply:
Can we test for the `:hover` selector instead?

Reply:
Ok - I've made that more flexible. Let me know what you think.